src/builders/browser/index.ts

```diff
@@ -17,42 +18,18 @@ export class SingleSpaBrowserBuilder extends BrowserBuilder {
     projectRoot: Path,
     host: virtualFs.Host<fs.Stats>,
     options: SingleSpaBuilderSchema,
-  ): Configuration {
-
-    const libraryName = options.libraryName || this.context.targetSpecifier && this.context.targetSpecifier.project;
-
+  ): Configuration {    
     // Disable es2015 polyfills
     // tslint:disable-next-line: max-line-length
     // https://github.com/angular/angular-cli/blob/3d8064bb64d72557474a7484f1b85eaf35788916/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts#L56
     options.es5BrowserSupport = false;
-
     // Generate Angular CLI's default Browser Webpack cofiguration
-    const config = super.buildWebpackConfig(root, projectRoot, host, options);
-
-    // Remove bundles
-    delete config.entry.polyfills;
-    delete config.entry.styles;
-    delete config.optimization.runtimeChunk;
-    delete config.optimization.splitChunks;
-
-    // Remove Angular's IndexHtmlWebpackPlugin from build
-    const indexHtmlWebpackPluginIndex = config.plugins.findIndex(
-      plugin => (plugin as any).constructor.name === 'IndexHtmlWebpackPlugin',
-    );
-
-    if (indexHtmlWebpackPluginIndex > -1) {
-      config.plugins.splice(indexHtmlWebpackPluginIndex, 1);
-    }
+    const config = super.buildWebpackConfig(root, projectRoot, host, options);    
 
-    return webpackMerge.smart(config, {
-      output: {
-        library: libraryName,
-        libraryTarget: options.libraryTarget as any,
-      },
-      externals: {
-        'zone.js': 'Zone',
-      },
-    });
+    // Delegate the building of the webpack config to the new builder.
+    // Builder based on custom builders implmented by @meltedspark
+    // https://github.com/meltedspark/angular-builders
+    return SingleSpaWebpackBuilder.buildWebpackConfig(root, options.singleSpaWebpackConfigPath, config, options, this.context);
```

Comment:
Can you add to [this part of the readme](https://github.com/CanopyTax/single-spa-angular#ng-build-options) documentation for this new option?